### ISSUE TITLE
Users having trouble with picking the year

### ISSUE THREAD

**Issue by boozedog** [dbushell/Pikaday#353]
For many users, it's not obvious that clicking the year lets them pick another year. So they're complaining they have to click the arrows more than 12 times to get to a prior year ...

In the short term, we're adding a note to the web page to instruct users about how to pick the year. But I wonder if something can be done to make it more obvious for those users who are not imaginative enough to try clicking on the year?

**Comment by timaschew** [comment]
+1
maybe a default hover style which gives a better feedback that it's clickable

**Comment by boozedog** [comment]
I think the problem with hover is then the end-user is expected to know that they can do something useful with the Month/Year. Better to have a visible indication without requiring hover, IMO.

Sad to see no response from devs on this item  :-(

**Comment by nathancahill** [comment]
Agreed. It might be good to have something like &blacktriangledown; beside the year.

**Comment by boozedog** [comment]
Agree 100%

**Comment by rikkert** [comment]
Nice, if anyone can start a branch on this that would be a great addition.

**Comment by nathancahill** [comment]
Just added a pull request @rikkert #380

**Comment by feeekkk** [comment]
This will insert a css dropdown arrow after the month and year
```
.pika-label:after {
	font-size: .75rem;
	padding: 0.25rem;
	content: '▼';
	opacity: 0.4;
}

.pika-label:hover:after {
	opacity: 1;
}
```
